### PR TITLE
feat(content): spanish translation

### PR DESCRIPTION
**Pull request by DanielGallo84**
### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: docs/es/README.md [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+    </figcapture>
+  </figure>
+  <figure>
+    <HopeIcon icon="fa-solid" />
+    <figcapture>
+      <h3>Veränderbar</h3>
+      Jede Person kann etwas zu ihrem Lieblingsprogramm beitragen, jederzeit. Ich kann das Programm so verwenden, wie ich es am liebsten will.
+    </figcapture>
+  </figure>
+</div> -->
```
Comment (mahula) [inline docs/es/README.md:231]: Please remove this leftver fr omcopying.

File: docs/es/contact/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+sidebar: false
+---
+
+## Developer Association
+
+busFaktor() e.V.  
+Wolfgang Huß, 2nd chairman  
+Haferstr. 5c  
+86179 Augsburg, Germany
+
```
Comment (mahula) [inline docs/es/contact/README.md:14]: Wolfgang Huß, 2º Presidente 
Haferstr. 5c  
86179 Augsburg, Alemania

**Review by mahula — CHANGES_REQUESTED** [review]
Thank you very much for your contribution.
Please change the two things mentioned.

File: docs/es/contact/README.md (hunk 2 above)
Comment (Tirokk) [reply docs/es/contact/README.md:14]: To get the Spanish version onto the website now I allow my self to take this as a note in the follow up issue #158 .

File: docs/es/README.md (hunk 1 above)
Comment (Tirokk) [reply docs/es/README.md:231]: To get the Spanish version onto the website now I allow my self to take this as a note in the follow up issue #158 .

**Review by Tirokk — APPROVED** [review]
Hey @DanielGallo84 

Really cool that you cared for the Spanish version of our `ocelot.social` project website.
😍💫

I approve even @mahula requested changes I agree with.
And I take this as notes in the follow up issue #158 .

**Review by Tirokk — APPROVED** [review]
Nice that you cared already for @mahula suggestions. 🙏🏼

I approve your work. 🚀🚀💫

**Pull request merged**